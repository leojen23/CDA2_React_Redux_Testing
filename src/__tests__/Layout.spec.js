import React from 'react'
import { shallow, mount } from 'enzyme';

import Layout from '../components/Layout';

import '../setUpTests';

jest.mock("../components/home", () => {
    return "home-component";
})
describe('Layout snapshot',()=>{
    it('It renders without crashing', () => {
        let wrapper = shallow(<Layout />);
        expect(wrapper);
    });
    it('rendercorrectly', ()=> {
        let wrapper = shallow(<Layout />);
        expect(wrapper).toMatchSnapshot();
    })

    it('It renders Home', () => {
        const wrapper = mount(<Layout />);
        let home = wrapper.find('home-component');
        expect(home.props.fetchRandomNumber)
    });
});
