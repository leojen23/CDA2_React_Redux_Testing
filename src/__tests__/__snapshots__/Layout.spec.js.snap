// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout snapshot rendercorrectly 1`] = `ShallowWrapper {}`;
